src/components/document-metadata-modal.js

```diff
@@ -138,25 +148,32 @@ function DocumentMetadataModal({
 
   const handleCancel = () => onClose();
 
+  const handleValuesChange = (_, { cloningStrategy, cloningTargetRoomId }) => {
+    const documentRoomId = determineDocumentRoomId({ mode, initialDocumentMetadata, documentToClone, cloningStrategy, cloningTargetRoomId });
+    if (documentRoomId) {
+      formRef.current.setFieldsValue({ allowedOpenContribution: DOCUMENT_ALLOWED_OPEN_CONTRIBUTION.metadataAndContent });
+    }
```

Comment:
Good idea to reset this value!
I think the case from`renderAllowedOpenContributionFormItem` (`if (documentRoomId || noTargetRoomSelectedYet)`) applies here too though, we'd need to reset also if `noTargetRoomSelectedYet`.
But I guess we can't avoid this duplication and have the form reset as part of the `renderAllowedOpenContributionFormItem`..?